data_managers/data_manager_sortmerna_database_downloader/data_manager/data_manager_sortmerna_download.py

```diff
@@ -126,23 +126,28 @@ def move_index_files(archive_content_path, target_dir, data_tables, version):
         if not filename.endswith("fasta"):
             continue
         input_filepath = os.path.join(file_dir, filename)
-        output_filepath = os.path.join(target_dir, filename)
-        # Move file
-        os.rename(input_filepath, output_filepath)
+        # Extract the db name
+        db_name = os.path.splitext(filename)[0]
+        # Create the directory where to put the fasta files and indexed files
+        filedir = os.path.join(target_dir, db_name)
+        os.mkdir(filedir)
+        fasta_filepath = os.path.join(filedir, "%s.fasta" % db_name)
+        indexed_filepath = os.path.join(filedir, db_name)
+        # Move the fasta file
+        os.rename(input_filepath, fasta_filepath)
```

Comment:
You should use [shutil.move](https://docs.python.org/2/library/shutil.html#shutil.move) because [os.rename](https://docs.python.org/2/library/os.html#os.rename) may not work across filesystems.